src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -1027,6 +1116,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
 
           // TODO use privilege API to filter instructors who has INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS
           // in the feedback session
+              // TODO change the instructorsCanBePreviewdAs in session edit to instructorsOfCourse
```

Comment:
Was this moved from somewhere else? Not really a fan of adding TODO's unless it's an uncompleted part of this PR (or deadline extensions series of PR) that will be added by you very soon. As you can see, once we leave a TODO there it's probably not going to be done anytime soon (the one above it is from 3 years ago), partly because these issues are not captured by our issue tracker.

On a side note, indention is off here and minor type in "Previewed".

Reply:
My intention was to nest this TODO to be under the other TODO. I realized that once the above TODO is fixed, it'll also unknowingly affect the modal. This is because the modal depends on having all the instructors of the course, while the TODO fix will filter some instructors out.
Hence, it's not a uncompleted part of our PR, but something for the fixer of the other TODO to consider.

Reply:
I see the issue now. This brings up a more pertinent issue, you shouldn't be using `instructorsCanBePreviewedAs` as the list of all instructors in the course. Could you refactor this to make it clearer?

Reply:
Good idea, I was initially hesitant, as it'll be touching outside the scope of extension. Your suggestion will make it clearer